src/js/git/index.js

```diff
@@ -3070,39 +3070,62 @@ var Tag = Ref.extend({
 
 function RevisionRange(engine, specifiers) {
```

Comment:
this doesnt have a ton of dependencies so maybe we could split it out one day, but no big deal.

Reply:
yeah makes sense